### ISSUE TITLE
Check box & radio button error labels with inline form validation - pointers in wrong direction.

### ISSUE THREAD

**Issue by VanquishedWombat** [fomantic/Fomantic-UI#1387]
# Bug Report
TL:DR - Checkbox (and radio button) with inline form validation has an issue with direction of label arrows when using grouped controls (vertical stack) . 

Long version: Form validation is a FUI behavior. Checkbox / radio button is a fui module. Form validation has a form-level option attribute named 'inline' which will, when set True, cause the form validation error messages to be shown inline, either underneath or to the right of the inputs to which they relate.  A common pattern is to have check boxes / radio buttons arranged in a vertical collection with a prompt at the top. See following image:

![image](https://user-images.githubusercontent.com/55594003/78039531-230af300-7366-11ea-9a1b-0e2389393fb7.png)

As can be seen, the issue here is that the inline error labels pointer is pointing upwards when it should be pointing to the left.

## Steps to reproduce
Reproduced by specific HTML formation - see test case link below.

## Expected result
With a collection of check boxes or radio buttons arranged in a vertical group, the pointers for inline error labels should point to the control with the validation error. Currently they point up. 

## Actual result
With a collection of check boxes or radio buttons arranged in a vertical group, the pointers for inline error labels point up. 

## Testcase

See codepen https://codepen.io/JEE42/pen/vYOPOBz

## Screenshot (if possible)

Provided above.

## Version
v 2.8.4 (confirmed with @lubber)

## Workaround
It seems that if the enclosing field div around the checkbox / radio is assigned class 'inline' then the error label pointer will be correctly pointing to the left. This seems non-semantic because the fields div is defined with the  'group' class to have the check boxes stack, therefore declaring the checkbox wrapping div 'inline' does not feel as if intended.  Overall wonder if it is an intention or a happy discovery. Either way, without confirmation, I worry whether I should deploy the workaround or if I should pursue a more formal solution.

**Comment by AdamHWeiss2017** [comment]
Place your inline tag in the outer "grouped fields" tag --> "grouped inline fields". All children will inherit. I played with the codepen and adding a padding-bottom to the checkbox class gives some nice spacing between items.

**Comment by VanquishedWombat** [comment]
Thanks @AdamHWeiss2017 . I concur that the 'grouped inline fields' approach solves the arrow direction. Also as you point out the bottom margin on the checkboxes in that CSS path needs an adjustment to avoid the inline error labels colliding. See codepen https://codepen.io/JEE42/pen/LYVavRZ for a working example. I added CSS

 .grouped.inline.fields .ui.radio.checkbox {
  margin-bottom: 0.4em;
}

to provide a reasonable spacing without going too crazy.

## Request

Add the above css into the Fomantic-UI source to make a permanent fix.

**Comment by lubber-de** [comment]
@VanquishedWombat I adopted your suggested margin adjustment in #1617 now
